pkg/lib/patternfly/patternfly-4-overrides.scss

```
@@ -71,7 +71,7 @@ svg {
 
   > .pf-c-card__actions {
       flex-wrap: wrap;
-      button {
+      button:not(.pf-c-dropdown__toggle){
```

```suggestion
      button:not(.pf-c-dropdown__toggle) {
```

Suuuuuuper minor: This should have a space before the curly bracket.

---

Done